rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -197,8 +199,100 @@ object Runtime {
         )
     }.sequence
 
+  object SystemProcess {
+    case class Context[F[_]: Async](
+        space: RhoISpace[F],
+        dispatcher: RhoDispatch[F],
+        registry: Registry[F],
+        shortLeashParams: ShortLeashParams[F],
+        blockTime: BlockTime[F]
+    ) {
+      val systemProcesses = SystemProcesses[F](dispatcher, space)
+    }
+
+    case class Definition[F[_]](
+        urn: String,
+        fixedChannel: Name,
+        arity: Arity,
+        bodyRef: BodyRef,
+        handler: Context[F] => (Seq[ListParWithRandomAndPhlos], Int) => F[Unit],
+        remainder: Remainder = None
+    ) {
+      def toDispatchTable(
+          context: SystemProcess.Context[F]
+      ): (BodyRef, (Seq[ListParWithRandomAndPhlos], Arity) => F[Unit]) =
+        bodyRef -> handler(context)
+
+      def toUrnMap: (String, Par) = {
+        val bundle: Par = Bundle(fixedChannel, writeFlag = true)
+        urn -> bundle
+      }
+
+      def toProcDefs: (Name, Arity, Remainder, BodyRef) =
+        (fixedChannel, arity, remainder, bodyRef)
+    }
+  }
+
+  val stdSystemProcesses: Seq[SystemProcess.Definition[Task]] = Seq(
```

Comment:
please make it a `def` parameterized with `F[_]`

Reply:
it requires a bigger refactoring because stdout, stdOutAck and others are returning Task[_]

Reply:
I've just double-checked that: both `systemProcesses` and `registry` are F-parameterized. This includes systemProcesses.stdoutAck. There's no reason to fix to Task here. I might missing sth though? Point me to it :)

Reply:
You are right. Idea was reporting the wrong type and I trusted it and didn't double check.